specification/machinelearningservices/AzureAI.Evaluation/model.tsp

```diff
@@ -133,11 +133,88 @@ model UpdateEvaluationRequest {
   description: string;
 }
 
-
 #suppress "@azure-tools/typespec-providerhub/no-inline-model" "Need to create reponses correctly"
 alias ResourceCreatedResponse<T extends TypeSpec.Reflection.Model> = TypeSpec.Http.Response<201> &
   T;
 
 #suppress "@azure-tools/typespec-providerhub/no-inline-model" "Need to create reponses correctly"
 alias OkResponse<T extends TypeSpec.Reflection.Model> = TypeSpec.Http.Response<200> &
   T;
+
+@doc("Frequency of the schedule - day, week, month, hour, minute")
+enum Frequency {
+    Month,
+    Week,
+    Day,
+    Hour,
+    Minute
+}
+
+@doc("WeekDay of the schedule - Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday")
+enum WeekDays {
+    Monday,
+    Tuesday,
+    Wednesday,
+    Thursday,
+    Friday,
+    Saturday,
+    Sunday
+}
+
+@doc("Recurrence Definition")
+model Recurrence {
+    @doc("The frequency to trigger schedule.")
+    frequency: Frequency;
+
+    @doc("Specifies schedule interval in conjunction with frequency")
+    interval: int32;
+
+    @doc("The recurrence schedule.")
+    schedule: RecurrenceSchedule;
+}
+
+@doc("RecurrenceSchedule Definition")
+model RecurrenceSchedule {
+    @doc("List of hours for the schedule.")
+    hours: int32[];
+
+    @doc("List of minutes for the schedule.")
+    minutes: int32[];
+
+    @doc("List of days for the schedule.")
+    weekDays: WeekDays[];
+
+    @doc("List of month days for the schedule")
+    monthDays: int32[];
+}
+
+@doc("SamplingStrategy Definition")
+model SamplingStrategy {
+    @doc("Sampling rate")
+    rate: float32;
+}
+
+@doc("Evaluation Schedule Definition")
+model EvaluationSchedule extends Evaluation {
+  @doc("Recurrence pattern for the evaluation.")
+  recurrence: Recurrence;

```

Comment:
Curious if this can be covered by cron schedule ? Recurrence seems too complicated to put together.

Reply:
can we add both for user convenience? main reason to stick with recurrence are 1. all users might not be well versed with cron expression and we already use similar convention in other schedule features like model monitoring.

For now, kept both, will take final call after the review.